runtime/src/main/java/org/corfudb/runtime/view/AbstractView.java

```diff
@@ -114,6 +115,8 @@ public Layout getCurrentLayout() {
                             + "invalidate view", we.getCorrectEpoch());
                 } else if (re instanceof NetworkException) {
                     log.warn("layoutHelper: System seems unavailable", re);
+                } else if (re instanceof ShutdownException) {
```

Comment:
Setting aside the TimeoutException, why can't we use catch clauses for catching and dealing with different exceptions (instead of manually checking the types)?